python/cuda_parallel/pyproject.toml

```diff
@@ -26,7 +26,13 @@ dynamic = ["version"]
 readme = { file = "README.md", content-type = "text/markdown" }
 
 [project.optional-dependencies]
-test = ["pytest", "pytest-xdist", "cupy-cuda12x", "typing_extensions"]
+test = [
+  "pytest",
+  "pytest-xdist",
+  "cupy-cuda12x",
+  "typing_extensions",
+  "pytest-benchmark",
```

Comment:
do we want `pytest-benchmark` in the test dep group? maybe start a different dependency group for benchmarking, again for future scaling reasons?

Reply:
I did consider this but it felt like a bit overkill. Also when in doubt, [do as RAPIDS does](https://github.com/rapidsai/cudf/blob/bd5167435fdf812941092856e71ff2adbdcbb328/python/cudf/pyproject.toml#L59)